cdp_scrapers/instances/seattle.py

```diff
@@ -72,14 +72,10 @@ def __init__(self):
         )
 
         try:
-            urlopen("https://seattlechannel.org/")
-        except URLError:
-            pass
-        else:
-            raise Exception(
-                "seattlechannel.org may have fixed their SSL cert. "
-                "Check and fix 'requests.get(*, verify=False)' calls"
-            )
+            url = "https://seattlechannel.org/"
+            urlopen(url)
+        except URLError as ue:
+            raise Exception(f"Unable to reach {url}: {ue}")
```

Comment:
Is this the right way to raise an exception? Any suggestions on error messages?

Reply:
This is perfectly suitable for our purposes.